src/data/data.js

```diff
@@ -13,7 +13,7 @@ const pageData = {
           'Promovemos la recolección del aceite usado de cocina, para que pueda ser utilizado nuevamente en una energía renovable.',
         button: {
           text: 'Registrate',
-          href: 'Register',
+          href: '/registro.html',
```

Comment:
Sorry @omeneses, could you please remove the extension `.html` from the links. It is not necessary anymore

Reply:
Done!!!